cmd/doctor.go

```diff
@@ -6,93 +6,129 @@ import (
 	"github.com/ForgeRock/forgeops-cli/internal/factory"
 	"github.com/ForgeRock/forgeops-cli/internal/printer"
 	"github.com/ForgeRock/forgeops-cli/pkg/doctor"
+	"github.com/ForgeRock/forgeops-cli/pkg/health"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-// cmd globals config
-var kubeConfig string
-var overrides = &clientcmd.ConfigOverrides{}
-var ctx context.Context
-var doctorFlags *genericclioptions.ConfigFlags
+var (
+	// cmd globals config
+	kubeConfig    string
+	overrides     = &clientcmd.ConfigOverrides{}
+	ctx           context.Context
+	doctorFlags   *genericclioptions.ConfigFlags
+	operatorFlags *genericclioptions.ConfigFlags
+	allNamespaces bool
 
-var ignoreProducts = []string{"ig"}
-
-var ds = &cobra.Command{
-	Use:   "directoryserver",
-	Short: "Check the status of Directory Server deployment",
-	Long: `
+	ds = &cobra.Command{
+		Use:   "directoryserver",
+		Short: "Check the status of Directory Server deployment",
+		Long: `
 	Check the status of Directory Server deployment by checking ready state and configuration.
 	  * check workload state
 	  * should we call an endpoint/ldap?
 	`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		printer.Warnln("Not implemented")
-		return nil
-	},
-	DisableAutoGenTag: true,
-}
+		RunE: func(cmd *cobra.Command, args []string) error {
+			printer.Warnln("Not implemented")
+			return nil
+		},
+		Hidden:            true,
```

Comment:
So is this one supposed to be hidden? Just checking.

Reply:
At the moment yes.